packages/components/psammead-story-promo/src/TextStyles.jsx

```diff
@@ -19,9 +19,7 @@ const fullWidthColumnsMaxScaleable = `100%`;
 
 const halfWidthColumnsMaxScaleable = `50%`;
 
-const paddingDir = ({ dir }) => `padding-${dir === 'rtl' ? 'left' : 'right'}`;
-
-const TextGridColumnsTopStory = css`
+const TextGridColumnsTopStory = `
```

Comment:
Did we do a visual comparison on storybook between this branch and the storybook on github?  Lot of changes here 👍